neo/Consensus/ConsensusService.cs

```diff
@@ -374,17 +362,17 @@ private void HandleRecoveryInCurrentView(RecoveryMessage message, Snapshot snap)
                     }
                 }
 
-                for (int i = 0; i < context.Validators.Length; i++)
+                for (ushort i = 0; i < context.Validators.Length; i++)
```

Comment:
Shouldn't this be int now?

Reply:
https://github.com/neo-project/neo/blob/8efb049aa9a052df3b5c5b4cd5bd0a4bb99ecd3c/neo/Consensus/ConsensusService.cs#L374-L375

I think it's okay to keep it `ushort`. Because in L375, the method `RegenerateSignedPayload` accepts `ushort` as its second parameter.

Reply:
Ah ok.